tfs/src/main/java/hudson/plugins/tfs/model/AbstractHookEvent.java

```diff
@@ -111,90 +157,10 @@ static JSONObject fromResponseContributors(final List<GitStatus.ResponseContribu
                 if (scmTriggerItem == null) {
                     continue;
                 }
-                for (final SCM scm : scmTriggerItem.getSCMs()) {

```

Comment:
I took another look at the GitHub plugin for reference, it does appear they they have some code which does something similar to this: